docs/content/docs/ops/state/checkpoints.md

```diff
@@ -35,6 +35,64 @@ the same semantics as a failure-free execution.
 See [Checkpointing]({{< ref "docs/dev/datastream/fault-tolerance/checkpointing" >}}) for how to enable and
 configure checkpoints for your program.
 
+## Checkpoint Storage
+
+When checkpointing is activated, such state is persisted upon checkpoints to guard against data loss and recover consistently.
+Where the state is persisted upon checkpoints depends on the chosen **Checkpoint Storage**.
```

Comment:
```suggestion
When checkpointing is enabled, managed state is persisted to ensure consistent recovery in case of failures.
Where the state is persisted during checkpointing depends on the chosen **Checkpoint Storage**.
```